### PR TITLE
Fix _MolFileBondEndPts after dativeBondsToHaptic.

### LINKED ISSUE #6252: DativeBondsToHaptic doesn't set _MolFileBondEndPts correctly.
**Describe the bug**
Round-tripping ferrocene through HapticBondsToDative then DativeBondsToHaptic produces incorrect _MolFileBondEndPts.

**To Reproduce**
```
from rdkit import Chem
from rdkit.Chem.Draw import rdMolDraw2D
from rdkit.Chem.Draw import IPythonConsole
from IPython.display import SVG
from rdkit.Chem import rdmolops

ferrocene = """ferrocene
     RDKit          2D

  0  0  0  0  0  0  0  0  0  0999 V3000
M  V30 BEGIN CTAB
M  V30 COUNTS 13 12 0 0 0
M  V30 BEGIN ATOM
M  V30 1 C -34.237900 13.261800 0.000000 0 CHG=-1
M  V30 2 C -35.057400 14.266900 0.000000 0
M  V30 3 C -37.855700 14.284400 0.000000 0
M  V30 4 C -38.759100 13.293300 0.000000 0
M  V30 5 C -36.552000 12.683200 0.000000 0
M  V30 6 C -34.152900 8.363300 0.000000 0 CHG=-1
M  V30 7 C -34.972400 9.368400 0.000000 0
M  V30 8 C -37.770600 9.385900 0.000000 0
M  V30 9 C -38.674100 8.394800 0.000000 0
M  V30 10 C -36.450300 7.767900 0.000000 0
M  V30 11 Fe -36.435100 10.904500 0.000000 0 CHG=2
M  V30 12 * -36.522400 13.604600 0.000000 0
M  V30 13 * -36.404100 8.656100 0.000000 0
M  V30 END ATOM
M  V30 BEGIN BOND
M  V30 1 9 12 11 ENDPTS=(5 2 3 4 5 1) ATTACH=ALL
M  V30 2 1 1 5
M  V30 3 2 4 5
M  V30 4 1 4 3
M  V30 5 2 2 3
M  V30 6 1 1 2
M  V30 7 1 6 10
M  V30 8 2 9 10
M  V30 9 1 9 8
M  V30 10 2 7 8
M  V30 11 1 6 7
M  V30 12 9 13 11 ENDPTS=(5 7 8 9 10 6) ATTACH=ALL
M  V30 END BOND
M  V30 END CTAB
M  END
"""
mol = Chem.MolFromMolBlock(ferrocene)
mol
nmol = rdmolops.HapticBondsToDative(mol)
bmol = rdmolops.DativeBondsToHaptic(nmol)

bond = mol.GetBondWithIdx(0)
print(f"original molecule : {bond.GetProp('_MolFileBondEndPts')}")
bond = mol.GetBondWithIdx(11)
print(f"original molecule : {bond.GetProp('_MolFileBondEndPts')}")
bond = bmol.GetBondWithIdx(10)
print(f"round-trip molecule : {bond.GetProp('_MolFileBondEndPts')}")
bond = bmol.GetBondWithIdx(11)
print(f"round-trip molecule : {bond.GetProp('_MolFileBondEndPts')}")
```

**Expected behavior**
Output is:

original molecule : (5 2 3 4 5 1)
original molecule : (5 7 8 9 10 6)
round-trip molecule : (1 2 0 3 4)
round-trip molecule : (6 7 5 8 9)

It should be
original molecule : (5 2 3 4 5 1)
original molecule : (5 7 8 9 10 6)
round-trip molecule : (5 2 3 1 4 5)
round-trip molecule : (5 7 8 6 9 10)

**Configuration (please complete the following information):**
 - RDKit version: current master, build from source.
 - OS: macOS Ventura 13.2.1

**Additional context**
This came up whilst writing the documentation - always a mistake :-).
Fix incoming.

### PR DESCRIPTION
**Pull request by DavidACosgrove**
#### Reference Issue
Fixes #6252 and #6253.

#### What does this implement/fix? Explain your changes.
Gets the format for the _MolFileBondEndPts correct when setting it.

### CODE REVIEW

File: Code/GraphMol/MolOps.cpp [hunk 1]
```diff
@@ -998,17 +998,20 @@ void addHapticBond(RWMol &mol, unsigned int metalIdx,
   unsigned int numbonds = mol.addBond(dummyIdx, metalIdx, Bond::DATIVE);
   auto bond = mol.getBondWithIdx(numbonds - 1);
 
-  // Get the atom numbers for the end points.
+  // Get the atom numbers for the end points.  First number is the
```
Comment (greglandrum) [inline Code/GraphMol/MolOps.cpp:1001]: Sorry, this should have been caught in the review of the initial PR

**Review by greglandrum — CHANGES_REQUESTED** [review]
the tests need help

File: Code/GraphMol/MolOps.cpp (hunk 1 above)
Comment (DavidACosgrove) [reply Code/GraphMol/MolOps.cpp:1001]: Yeah, but given that I'd just written my own parser for it, I might have been expected to get it right the first time!

**Review by greglandrum — APPROVED** [review]
LGTM

**Pull request merged**